### PR TITLE
Feature/tutorial layout merge fix

### PR DESCRIPTION
**Pull request by joel-hamill**
## Description
This PR replaces https://github.com/dcos/dcos-docs/pull/930/.
- https://jira.mesosphere.com/browse/DCOS_OSS-654
- TODO merge redirects: dcos/dcos-website#1140
- [x] TODO: make changes for 1.9

## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by joel-hamill** [comment]
@sascala i have merged in the content from https://dcos.io/docs/1.8/usage/marathon/ to https://dcos.io/docs/1.8/usage/managing-services/. Please review and let me know what you think (I recommend pulling this PR and building locally to see changes inline).

File: 1.8/usage/managing-services/deploy-docker-app.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
-post_title: Deploying a Docker-based Service to Marathon
```
Comment (sascala) [inline 1.8/usage/managing-services/deploy-docker-app.md]: I'd move this to underneath the "application basics" page

File: 1.8/usage/managing-services/expose-service.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
 ---
-post_title: Deploying Public Services
-nav_title: Public Services
-menu_order: 10
+post_title: Exposing a Service
+menu_order: 005
 ---
 
-DC/OS agent nodes can be designated as [public](/docs/1.8/overview/concepts/#public) or [private](/docs/1.8/overview/concepts/#private) during [installation](/docs/1.8/administration/installing/). Public agent nodes provide public access to your DC/OS applications. By default apps are launched on private agent nodes. 
+DC/OS agent nodes can be designated as [public](/docs/1.8/overview/concepts/#public) or [private](/docs/1.8/overview/concepts/#private) during [installation](/docs/1.8/administration/installing/). Public agent nodes provide access from outside of the cluster via infrastructure networking to your DC/OS services. By default apps are launched on private agent nodes and are not accessible.
```
Comment (sascala) [inline 1.8/usage/managing-services/expose-service.md]: Suggest, "By default, services are launched on private agent nodes and are not accessible from outside the cluster."

File: 1.8/usage/managing-services/expose-service.md [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-menu_order: 10
+post_title: Exposing a Service
+menu_order: 005
 ---
 
-DC/OS agent nodes can be designated as [public](/docs/1.8/overview/concepts/#public) or [private](/docs/1.8/overview/concepts/#private) during [installation](/docs/1.8/administration/installing/). Public agent nodes provide public access to your DC/OS applications. By default apps are launched on private agent nodes. 
+DC/OS agent nodes can be designated as [public](/docs/1.8/overview/concepts/#public) or [private](/docs/1.8/overview/concepts/#private) during [installation](/docs/1.8/administration/installing/). Public agent nodes provide access from outside of the cluster via infrastructure networking to your DC/OS services. By default apps are launched on private agent nodes and are not accessible. 
 
-To launch an app on a public node, you must create a Marathon app definition with the `"acceptedResourceRoles":["slave_public"]` parameter specified.
+To launch an app on a public node, you must create a Marathon app definition with the `"acceptedResourceRoles":["slave_public"]` parameter specified and configure an edge load balancer and service discovery mechanism. This example
```
Comment (sascala) [inline 1.8/usage/managing-services/expose-service.md]: Suggest: "To launch a service on a public node, create a Marathon app definition..."

File: 1.8/usage/managing-services/index.md [hunk 4]
```diff
@@ -1,6 +1,19 @@
 ---
-post_title: Managing Universe Services
+post_title: Managing Services
 menu_order: 70 
 ---
 
-DC/OS Universe services are Marathon applications that are packaged and available from the DC/OS Universe repository. Available DC/OS services include [Mesos frameworks](http://mesos.apache.org/documentation/latest/frameworks/) and other applications. DC/OS services are created by the DC/OS [community](https://dcos.io/community/) and by Mesosphere. 
+DC/OS uses Marathon to manage processes and services and is the "init system" for DC/OS. Marathon starts and monitors your applications and services, automatically healing failures.
```
Comment (sascala) [inline 1.8/usage/managing-services/index.md]: Suggest: "DC/OS uses Marathon to manage processes and services: Marathon is the "init" system for DC/OS.

File: 1.8/usage/managing-services/index.md [hunk 5]
```diff
@@ -1,6 +1,19 @@
...
+
+A native Marathon instance is installed as a part of DC/OS installation. After DC/OS has started, you can manage the native Marathon instance through the **Services** tab of the DC/OS web interface or from the DC/OS CLI with the `dcos marathon` command.
+
+DC/OS services are Marathon applications that are deployed on DC/OS. DC/OS services are available from a package repository such as the [Mesosphere Universe](/docs/1.8/overview/concepts/#mesosphere-universe), or you can create your own.
```
Comment (sascala) [inline 1.8/usage/managing-services/index.md]: comma after "repository"

File: 1.8/usage/managing-services/index.md [hunk 6]
```diff
@@ -1,6 +1,19 @@
...
+
+#  DC/OS Services
+
+You can run DC/OS services you create or install a package from the [Universe package repository](/docs/1.8/usage/webinterface/#-a-name-universe-a-universe). Both services you create and those you install from Universe appear on the **Services** tab of the DC/OS web interface when they are running.
+
+Services you create yourself are administered by Marathon and can be configured and run [from the DC/OS CLI](/docs/1.8/usage/cli/command-reference/) with `dcos marathon` subcommands (eg `dcos marathon app add myapp.json`) or via the DC/OS web interface.
```
Comment (sascala) [inline 1.8/usage/managing-services/index.md]: `<myapp>.json`

File: 1.8/usage/managing-services/install.md [hunk 7]
```diff
@@ -1,41 +1,83 @@
 ---
-post_title: Installing Services
-nav_title: Installing
+post_title: Deploying Services
+nav_title: Deploying
 menu_order: 000
 ---
+
+You can install and deploy DC/OS services from a package repository such as the [Mesosphere Universe](/docs/1.8/overview/concepts/#mesosphere-universe) or deploy your own user-created services.
```
Comment (sascala) [inline 1.8/usage/managing-services/install.md]: comma after "repository"

File: 1.8/usage/managing-services/install.md [hunk 8]
```diff
@@ -1,41 +1,83 @@
...
 For example, you would use the following command to install Chronos with the default parameters.
     
 ```bash
 $ dcos package install chronos
 ```
     
-## Installing a service using the UI
+## Web interface
+
+From the DC/OS web interface you can install services from the **Services** or **Universe** tab. The Universe tab shows all of the available DC/OS services from package [repositories](/docs/1.8/usage/repo/). The Services tab provides a full-featured interface to the native DC/OS Marathon instance.
```
Comment (sascala) [inline 1.8/usage/managing-services/install.md]: comma after "interface".

File: 1.8/usage/managing-services/install.md [hunk 9]
```diff
@@ -1,41 +1,83 @@
...
+
+```bash
+$ dcos package list
+```
+
+### Web interface
+
+Go to the **Services** tab and confirm that the service is running. For more information, see the web interface [documentation](/docs/1.8/usage/webinterface/#services).
+
+**Tip:** Some services from the "Community Packages" section of the Universe will not show up in the DC/OS service listing. For these, inspect the service's Marathon app in the Marathon web interface to verify that the service is running and healthy.
```
Comment (sascala) [inline 1.8/usage/managing-services/install.md]: Maybe let users know that they can get to the Marathon UI at `<dcos-url>/#marathon`?

File: 1.8/usage/managing-services/install.md [hunk 10]
```diff
@@ -1,41 +1,83 @@
...
 
-### Universe tab
+## CLI
 
-1.  Navigate to the [**Universe**](/docs/1.8/usage/webinterface/#universe) page in the DC/OS UI.
+The general syntax for installing a service with the CLI follows. 
 
-2.  Choose your package and click **Install package**. 
+```bash
+$ dcos marathon app add <myapp.json>
```
Comment (sascala) [inline 1.8/usage/managing-services/install.md]: `<myapp>.json` (the `.json` part isn't a variable)

File: 1.8/usage/managing-services/install.md [hunk 11]
```diff
@@ -44,11 +86,135 @@ From the DC/OS UI you can install services from the **Services** or **Universe**
...
-
-**Tip:** Some services from the "Community Packages" section of the Universe will not show up in the DC/OS service listing. For these, inspect the service's Marathon app in the Marathon UI to verify that the service is running and healthy.
\ No newline at end of file
+### Web interface
+
+Go to the **Services** tab and confirm that the service is running. For more information, see the web interface [documentation](/docs/1.8/usage/webinterface/#services).
+
+# Example
+
+This tutorials shows how to create and deploy a simple one-command service and a containerized service using both the DC/OS web interface and the CLI.
```
Comment (sascala) [inline 1.8/usage/managing-services/install.md]: "This tutorial"

File: 1.8/usage/managing-services/scale-service.md [hunk 12]
```diff
@@ -0,0 +1,25 @@
+---
+post_title: Scaling a Service
+nav_title: Scaling
+menu_order: 003.2
+---
+
+This tutorials shows how to scale a service using the DC/OS web interface and the CLI.
```
Comment (sascala) [inline 1.8/usage/managing-services/scale-service.md]: "This tutorial"

File: 1.8/usage/managing-services/uninstall.md [hunk 13]
```diff
@@ -1,16 +1,16 @@
...
 post_title: Uninstalling Services
 nav_title: Uninstalling
-menu_order: 004
+menu_order: 007
 ---
 
 ## About uninstalling services
 
-Services can be uninstalled from either the web interface or the CLI. If the service has any reserved resources, you also need to run the framework cleaner script. The framework cleaner script removes the service instance from ZooKeeper, along with any data associated with it.  
+Services can be uninstalled from either the web interface or the CLI. If a Universe service has any reserved resources, you also need to run the framework cleaner script. The framework cleaner script removes the service instance from ZooKeeper, along with any data associated with it.
```
Comment (sascala) [inline 1.8/usage/managing-services/uninstall.md]: link to framework cleaner anchor here?

File: 1.8/usage/tutorials/create-service.md [hunk 14]
```diff
@@ -1,20 +1,10 @@
 ---
-post_title: Creating and Running a Service on DC/OS
-nav_title: Creating and Running a Service 
-menu_order: 4.5 
+post_title: Creating and Running a Service
+nav_title: Running
```
Comment (sascala) [inline 1.8/usage/tutorials/create-service.md]: Maybe "Running a Service"? "Running" seems a little vague

File: 1.8/usage/tutorials/create-service.md [hunk 15]
```diff
@@ -1,20 +1,10 @@
...
-
-Services you create yourself are administered by Marathon and can be configured and run [from the DC/OS CLI](/docs/1.8/usage/cli/command-reference/) with `dcos marathon` subcommands (eg `dcos marathon app add myapp.json`) or via the DC/OS web interface.
-
-# Universe Package Repository
-Packaged DC/OS services created by Mesosphere or the community, like Spark or Kafka, appear on the **Universe** tab of the DC/OS web interface, or you can search for a package from [the DC/OS CLI](/docs/1.8/usage/cli/command-reference/). You can configure and run Universe services from the DC/OS web interface or via the DC/OS CLI with the `dcos package install <package-name>` command. Visit the [Managing Services](/docs/1.8/usage/managing-services/) section to learn more about installing, configuring, and uninstalling services in Universe.
-
-# Create and Run Your Own DC/OS Service
-In this tutorial, we'll go over creating a simple one-command service and a containerized service using both the DC/OS web interface and the CLI.
+This tutorials shows how to create and deploy a simple one-command service and a containerized service using both the DC/OS web interface and the CLI.
```
Comment (sascala) [inline 1.8/usage/tutorials/create-service.md]: "This tutorial"

File: 1.8/usage/tutorials/index.md [hunk 16]
```diff
@@ -5,50 +5,68 @@ menu_order: 90
...
-
-Combined services and miscellaneous: 
-
-- [WordPress](https://github.com/dcos/examples/tree/master/1.8/wordpress/)
-- [Scale](https://github.com/dcos/examples/tree/master/1.8/scale), on-demand, near real-time, automated processing of large datasets
-- [Building an IoT Pipeline](/docs/1.8/usage/tutorials/iot_pipeline/)
-- [Bookkeeper](https://github.com/dcos/examples/tree/master/1.8/bookkeeper)
+# DC/OS examples repository
+
+The tutorials that are located in [dcos/examples](https://github.com/dcos/examples/) GitHub repo. These tutorials contain examples of how to install and use DC/OS services.
```
Comment (sascala) [inline 1.8/usage/tutorials/index.md]: ^ The first sentence is incomplete

**Review by sascala — COMMENTED** [review]
I have some edits, but in general I think the architectural changes you made make sense

File: 1.8/usage/managing-services/install.md (hunk 9 above)
Comment (joel-hamill) [inline 1.8/usage/managing-services/install.md]: I think the Services tab is the Marathon UI as far as DC/OS is concerned. Is there some functionality that is still missing?

**Pull request merged**